cmd/minishift/cmd/util/oc.go

```diff
@@ -102,18 +103,19 @@ func SetOcContext(profileName string) error {
 func RemoveCurrentContext() error {
 	ocPath := minishiftConfig.InstanceConfig.OcPath
 	cmd := "config unset current-context"
+	errBuffer := new(bytes.Buffer)
 
 	ocRunner, err := oc.NewOcRunner(ocPath, filepath.Join(constants.Minipath, "machines", constants.MachineName+"_kubeconfig"))
 	if err != nil {
 		if glog.V(2) {
 			fmt.Println(fmt.Sprintf("%s", err.Error()))
 		}
-		return errors.New(fmt.Sprintf("Error unsetting current-context"))
+		return errors.New(fmt.Sprint("Error unsetting current-context"))
```

Comment:
Here the fmt.Sprint() is not necessary.

Reply:
No, i thought it was but its not. Taking it out.